README.md

```diff
@@ -115,6 +115,13 @@ Requirements:
 - The provided DID MUST be compliant with the `did:itn` method.
 - The provided DID document MUST be compliant with the DID V1.0 DID document specification.
 
+Here is how create operation works:
+
+- The create DID request is parsed by the ITN agent, which validates its syntax as well as the value of each field to follow specific rules.
+- The ITN agent generates the DID identifier based on the `did:itn` method specification, then the network stores the Hash(DID) and Hash(DID Document) in a smart contract on all the utilized ledger (one private and one public)
```

Comment:
DID identifier is generated by the sender, not by ITN Agent.

Reply:
Updated.